### PR TITLE
feat(icon): add white Guarantee icon

### PR DESCRIPTION
**Pull request by tam-kis**
Removed white Guarantee icon. Using only one Guarantee icon with color props.
Screenshot 2022-05-04 at 14 20 03

### CODE REVIEW

**Comment by dzucconi** [comment]
these shouldn't have any color. they should be fillable by any system color.

File: packages/palette/src/svgs/GuaranteeWhiteIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+/** GuaranteeWhiteIcon */
+export const GuaranteeWhiteIcon: React.FC<IconProps> = ({
+  title = "Guarantee White",
+  ...props
+}) => {
+  return (
+    <Icon {...props} viewBox="0 0 18 18">
+      <Title>{title}</Title>
+      <Path
+        fill="#fff"
```
Comment (dzucconi) [inline packages/palette/src/svgs/GuaranteeWhiteIcon.tsx:13]: see other icons for example. should get rid of the other version too

**Review by dzucconi — CHANGES_REQUESTED** [review]

File: packages/palette/src/svgs/GuaranteeWhiteIcon.tsx (hunk 1 above)
Comment (tam-kis) [reply packages/palette/src/svgs/GuaranteeWhiteIcon.tsx:13]: You mean `color(props.fill!)` , it says `color` is deprecated? Also the other one has 2 paths of 2 different colors.

**Comment by dzucconi** [comment]
its fine for this purpose (not really but we dont have a choice until i rebuild the icon pipeline). u can add a class to the accent and target it via a styled tag

File: packages/palette/src/svgs/GuaranteeWhiteIcon.tsx (hunk 1 above)
Comment (tam-kis) [reply packages/palette/src/svgs/GuaranteeWhiteIcon.tsx]: @dzucconi I've updated it to use one Guarantee icon with the color props.

**Review by dzucconi — APPROVED** [review]

File: packages/palette/src/svgs/GuaranteeWhiteIcon.tsx (hunk 1 above)
Comment (dzucconi) [reply packages/palette/src/svgs/GuaranteeWhiteIcon.tsx]: Thanks!

**Comment by artsyit** [comment]
:rocket: PR was released in [`@artsy/palette-charts@18.6.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette-charts%4018.6.0), [`@artsy/palette@19.6.0`](https://github.com/artsy/palette/releases/tag/%40artsy/palette%4019.6.0) :rocket:

**Pull request merged**